### ISSUE TITLE
[vm/macos] ld: warning: object file was built for newer macOS version (10.15) than being linked (10.13)

### ISSUE THREAD

**Issue by dcharkes** [dart-lang/sdk#47834]
It is probably harmless, but it is noisy.

```
[3079/3135] LINK ./dart
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(memory.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(mutex.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(condition_variable.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(condition_variable_destructor.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(mutex_destructor.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(string.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(private_typeinfo.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(stdlib_new_delete.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_demangle.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_guard.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_virtual.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(UnwindLevel1.c.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(stdexcept.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(stdlib_stdexcept.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(thread.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(system_error.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(charconv.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(stdlib_exception.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(stdlib_typeinfo.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_handlers.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_exception.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_personality.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(abort_message.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(fallback_malloc.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(vector.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(future.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_exception_storage.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(cxa_default_handlers.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(libunwind.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(UnwindRegistersSave.S.o)) was built for newer macOS version (10.15) than being linked (10.13)
ld: warning: object file (../../buildtools/mac-x64/clang/bin/../lib/libc++.a(exception.cpp.o)) was built for newer macOS version (10.15) than being linked (10.13)
```

cc @athomas

**Comment by athomas** [comment]
I'm going to close this as a duplicate of https://github.com/dart-lang/sdk/issues/47820.

We need to upgrade the min macOS SDK version which currently is blocked on that issue.
